src/st/sel/sel.h

```diff
@@ -63,17 +63,17 @@ typedef struct {
 typedef struct {
     /* 801BC8E0 */ u32 icon[BLOCK_PER_CARD];
     /* 801BC91C */ u32 slot[BLOCK_PER_CARD];
-    /* 801BC958 */ u32 place[BLOCK_PER_CARD];
-    /* 801BC994 */ u32 unk_1[BLOCK_PER_CARD];
-    /* 801BC9D0 */ u32 unk_2[BLOCK_PER_CARD];
+    /* 801BC958 */ u32 stage[BLOCK_PER_CARD];
+    /* 801BC994 */ u32 roomX[BLOCK_PER_CARD];
+    /* 801BC9D0 */ u32 roomY[BLOCK_PER_CARD];
     /* 801BCA0C */ u32 level[BLOCK_PER_CARD];
-    /* 801BCA48 */ u32 money[BLOCK_PER_CARD];
-    /* 801BCA84 */ u32 percentage[BLOCK_PER_CARD];
+    /* 801BCA48 */ u32 gold[BLOCK_PER_CARD];
+    /* 801BCA84 */ u32 nRoomsExplored[BLOCK_PER_CARD];
     /* 801BCAC0 */ u32 playHours[BLOCK_PER_CARD];
     /* 801BCAFC */ u32 playSeconds[BLOCK_PER_CARD];
     /* 801BCB38 */ u32 playMinutes[BLOCK_PER_CARD];
     /* 801BCB74 */ u32 kind[BLOCK_PER_CARD]; // 0: play, 1: clear, 2: replay
-    /* 801BCBB0 */ u32 isRichter[BLOCK_PER_CARD];
+    /* 801BCBB0 */ u32 character[BLOCK_PER_CARD];
```

Comment:
It seems to me that if we want to rename to character we should use an `CHARACTER_RICHTER` enum here or similar? https://github.com/Xeeynamo/sotn-decomp/blob/57d47163ae0e093f97c0b6f8919a09a4a5c397ac/src/st/sel/2C048.c#L418